waiter/src/waiter/scheduler/marathon.clj

```diff
@@ -275,7 +275,8 @@
      :cpus cpus
      :healthChecks [{:protocol (-> (or health-check-proto backend-proto) hu/backend-proto->scheme str/upper-case)
                      :path health-check-url
-                     :gracePeriodSeconds grace-period-secs
+                     ;; configure grace period even when it is 0 as the value defaults to 300 when unspecified
+                     :gracePeriodSeconds (max grace-period-secs 1)
```

Comment:
I was thinking this would be a really large value if 0 to achieve the same effect.  Maybe something like `instance-expiry-mins*60`.  What do you think?

Reply:
Hmm, sounds good. Will update.

Reply:
Done.